README.md

```diff
@@ -0,0 +1,251 @@
+# DT-DIGITAX-API
+
+DigiTax is the #1 tax preparation software in the world to file taxes online. Easily file federal and state
+income tax returns with 100% accurate Expert Approved Guarantee. 
+
+[![GitHub issues](https://img.shields.io/github/issues/Leroyal/DT-DIGITAX-API)](https://github.com/Leroyal/DT-DIGITAX-API/issues)
+
+## Table of Contents
+
+* [RESTful Web Services](#rest)
+* [Technologies](#technologies)
+    * [Java](#java)    
+    * [Sprint Boot](#spring-boot)
+* <details>
+    <summary>DIGITAX API</summary>
+
+    * [Sign Up](#sign-up)
```

Comment:
@Jayanta-Redapple As you develop more APIs, also include them to the README. Consider updating the README as part of your standard coding. This way, everyone (devs/managers/stakeholders) is on the same page about how everything works.